src/Resources/app/storefront/package.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
I think the `README.md` still contains information about running `npm install`, can we remove this?